tutorials/quickstart-node.md

```diff
@@ -0,0 +1,497 @@
+
+Let's create this migration to modify the `page_loads` table and create a
+hypertable by first running the following command:
+
+```bash
+npx sequelize migration:generate --name add_hypertable
+```
+
+You should get a result that looks like this:
+
+```bash
+Sequelize CLI [Node: 12.16.2, CLI: 5.5.1, ORM: 5.21.11]
+
+migrations folder at [some_path] already exists.
+New migration was created at [some_path]/]20200601202912-add_hypertable.js .
+```
+
+And there should now be a file in your `migrations` folder that we can modify
+to look like the following:
+
+```js
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.query("SELECT create_hypertable('page_loads', 'time');");
+  },
+
+  down: (queryInterface, Sequelize) => {
+  }
+};
+```
+
+Now run the migration command from the command-line:
+
+```bash
+npx sequelize db:migrate
+```
+
+You should get the following result:
+
+```bash
+Sequelize CLI [Node: 12.16.2, CLI: 5.5.1, ORM: 5.21.11]
+
+Loaded configuration file "config/config.json".
+Using environment "development".
+== 20200601202912-add_hypertable: migrating =======
+== 20200601202912-add_hypertable: migrated (0.426s)
+```
+
+### Insert rows into TimescaleDB [](insert_rows)
+
+Now you have a working connection to your database, a table configured with 
+the proper schema, and a hypertable created to more efficiently query data 
+by time. Let's add data to the table.
+
+In the `index.js` file, we will modify the `/` route like so to first get the 
+`user-agent` from the request object (`req`) and the current timestamp. Then,
+we will save call the `create` method on our model (`PageLoads`), supplying
+the user agent and timestamp parameters:
+
+```javascript
+app.get('/', async (req, res) => {
+    // get the user agent and current time
+    const userAgent = req.get('user-agent');
+    var date = new Date();
+    var time = date.getTime();
+
+    try {
+        // insert the record
+        await PageLoads.create({
+            userAgent, time
+        });
+
+        res.send('Inserted!);
+    } catch (e) {
+        console.log('Error inserting data', e)
+    }
+})
+```
+
+### Execute a query [](execute_query)
+
+Each time the page is reloaded, we also want to display all information 
+currently in the table.
+
+To do this, we will once again modify the `/` route in our `index.js` file
+to call the Sequelize `findAll` function and retrieve all data from the
+`page_loads` table via the `PageLoads` model, like so:
+
+```javascript
+app.get('/', async (req, res) => {
+    // get the user agent and current time
+    const userAgent = req.get('user-agent');
+    var date = new Date();
+    var time = date.getTime();
+
+    try {
+        // insert the record
+        await PageLoads.create({
+            userAgent, time
+        });
+
+        // now display everything in the table
+        var message = await PageLoads.findAll();
```

Comment:
Should be `const` and want the variable pural because its getting multiple results.
`const messages = await PageLoads.findAll();`